command/command_testonly/server_testonly_test.go

```diff
@@ -37,12 +37,12 @@ const (
 	`
 	requestLimiterDisableHCL = `
   request_limiter {
-	disable = true
+	enable = false
   }
 `
 	requestLimiterEnableHCL = `
   request_limiter {
-	disable = false
```

Comment:
Thinking ahead to the time when we have this feature stabilized, we want it to be enabled by default for everyone, right? In my head, it would be easier to leave the config option as `disable`, since we will want it enabled, and will want to make people actively disable it if they don't want it, and the default if nothing is specified is `disable = false`. But, for this release, just default that option to be true. Otherwise, if the config option is `enable` and no one specifies anything, the default will be `enable = false`, which isn't what we want long term, right? I feel like changing the config option from `enable` to `disable` down the road is going to be a pain, with deprecations, etc.

Let me know if I'm misunderstanding this or overthinking it though.

Reply:
Yeah, this is a good point. So the argument goes something like "we prefer making default values mean something (i.e. bool = false by default), and it's more tolerable to accept that for a short period of time than it is to have that be the behavior when the feature is stable enough to be on by default"?

I personally prefer the `enable` (with default value being false), for now. I'm less certain about how much of a pain changing the configuration will be in the future, but I'd have to think about it more through the eyes of an operator.

Reply:
One thing to consider: there are some features which are enabled by default (automated reporting being one), where we override the default behavior (in other words, the default is `true`). That's not to say that this is the best UX, but just that there's some prior art at least.

Reply:
Maybe it should be something like `enable_experimental`?  Then we can replace that with `disable` later once it's no longer experimental.

Reply:
I'm not opposed to this idea. I think it'd potentially be quite useful to have a self-documenting config name like that. @raskchanky curious what you think about this ^